Guides.md

```diff
@@ -43,7 +53,8 @@ On OS X you will need sshfs installed. The easiest way to install sshfs is to in
 After that you can do:
 
 	$ mkdir CYD-remote
-	$ sshfs <liu-id>@ssh.cyd.liu.se:/cyd/home/<liu-id> CYD-remote
+	$ sshfs <liu-id>@ssh.cyd.liu.se: CYD-remote
```

Comment:
Ser till att ändra alla 😊

> 5 juni 2016 kl. 00:55 skrev Rovanion Luckey <notifications@github.com>:
> 
> In Guides.md:
> 
> > @@ -43,7 +53,8 @@ On OS X you will need sshfs installed. The easiest way to install sshfs is to in
> >  After that you can do:
> >  
> >  	$ mkdir CYD-remote
> > -	$ sshfs <liu-id>@ssh.cyd.liu.se:/cyd/home/<liu-id> CYD-remote
> > +	$ sshfs <liu-id>@ssh.cyd.liu.se: CYD-remote
> Denna ändring gör sökvägen olik den för de andra plattformarna. Hurså?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>